### ISSUE TITLE
GET /v1/volumes/<volname> doesn't display volume options set through volume options API

### ISSUE THREAD

**Issue by atinmu** [gluster/glusterd2#366]
While I was trying to set some volume options through volume options API, the command succeeded. However get volume API returned blank options field.

**Comment by aravindavk** [comment]
Do we have `GET /v1/volumes/:volname/options` implemented? Are you referring `GET /v1/volumes/:volname`?

**Comment by atinmu** [comment]
I'm referring to the latter i.e. `GET /v1/volumes/:volname`

**Comment by prashanthpai** [comment]
@atinmu This works for me. Could you please recheck ? There's my sample run:

```sh
$ curl -i -X POST http://127.0.0.1:24007/v1/volumes/test/options --data @options.json -H 'Content-Type: application/json'
HTTP/1.1 200 OK
Content-Type: application/json; charset=UTF-8
X-Request-Id: b9e416f0-a367-47fe-93ea-17a77a4d52a1
Date: Thu, 14 Sep 2017 14:18:10 GMT
Content-Length: 24

{"afr.eager-lock":"on"}
```

```sh
$ curl -s -X GET http://127.0.0.1:24007/v1/volumes/test | python -m json.tool
{
    "Auth": {
        "Password": "ef15aa7b-0e58-4e40-bcb3-5d8b9460522d",
        "Username": "8641ba36-bdd1-4afa-931f-8185f033349f"
    },
    "Bricks": [
        {
            "Hostname": "127.0.0.1",
            "NodeID": "e09eb636-84eb-4899-bc3b-588e937425dd",
            "Path": "/export/brick1/data",
            "VolumeID": "6a6f55df-6866-4312-8c90-bfeab26fd013",
            "VolumeName": "test"
        },
        {
            "Hostname": "127.0.0.1",
            "NodeID": "e09eb636-84eb-4899-bc3b-588e937425dd",
            "Path": "/export/brick2/data",
            "VolumeID": "6a6f55df-6866-4312-8c90-bfeab26fd013",
            "VolumeName": "test"
        }
    ],
    "Checksum": 0,
    "DistCount": 2,
    "ID": "6a6f55df-6866-4312-8c90-bfeab26fd013",
    "Name": "test",
    "Options": {
        "afr.eager-lock": "on"
    },
    "ReplicaCount": 1,
    "Status": 2,
    "Transport": "tcp",
    "Type": 0,
    "Version": 0
}
```

**Comment by prashanthpai** [comment]
Closing this as it's been fixed in PR #374
